src/experimental/safe_html_transform/profiles/default/controlpanel.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
```

Comment:
I don't think this is necessary here. You are registering an non-existent safe_html_transform control panel. we are re-using the @@filter-controlpanel.